src/test/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassDataAbstractionCouplingCheckTest.java

```diff
@@ -60,6 +60,52 @@ public void test() throws Exception {
     }
 
     @Test
+    public void testExludedPackageConcretePackages() throws Exception {
+        final DefaultConfiguration checkConfig =
+            createCheckConfig(ClassDataAbstractionCouplingCheck.class);
+
+        checkConfig.addAttribute("max", "0");
+        checkConfig.addAttribute("excludedPackages",
+            "com.puppycrawl.tools.checkstyle.checks.metrics.inputs.c,"
+                + "com.puppycrawl.tools.checkstyle.checks.metrics.inputs.b");
+
+        final String[] expected = {
+            "8:1: " + getCheckMessage(MSG_KEY, 2, 0, "[AAClass, ABClass]"),
+        };
+
+        verify(checkConfig, getPath("InputClassCouplingExcludedPackages.java"), expected);
```

Comment:
2 separate checks shouldn't use the same input. It makes one dependent on the other when we make changes to the input file for one test. We are currently modifying our tests to prevent this.
Input file should be duplicated with unique name.